### PR TITLE
Init agregator database from BDD

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Init agregator database from BDD

## Type of change

- Non-breaking change in test steps implementation

## Testing steps

N/A

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary

### CODE REVIEW

File: features/environment.py [hunk 1]
```diff
@@ -31,7 +38,8 @@ def before_scenario(context, scenario):
         return
 
 
-def clean_db(context, database="test"):
+def prepared_db(context, setup_files=CLEANUP_FILES, database="test"):
```
Comment (epapbak) [inline features/environment.py:41]: `prepare_db` instead of `prepare**d**_db`?

**Review by epapbak — APPROVED** [review]
+1

File: features/environment.py (hunk 1 above)
Comment (tisnik) [reply features/environment.py:41]: yup, you are right, lemme fix it

Comment (tisnik) [reply features/environment.py]: fixed

**Review by juandspy — APPROVED** [review]

**Pull request merged**